docs/building.md

```diff
@@ -1,7 +1,23 @@
-## Building the repository from source
+## Building the Repository from Source
 
-**TODO**
+The Toltec repository is automatically built for each commit on the `stable` and `testing` branches.
+You may want to build the repository locally when working with a new package recipe or to make sure that nobody tampered with the automated build system.
 
-to build all the packages, run `make` from the base repository. this will
-involve downloading a docker image (1GB) and the remarkable toolchain which
-will expand to 3GB. the final build artifacts will be found in `artifacts/`.
+To proceed, create a local clone of the Git repository.
+
+```sh
+git clone https://github.com/toltec-dev/toltec
+```
+
+You’ll be on the `stable` branch by default.
+If you want to build the `testing` branch, switch to it manually.
+Before running the build, make sure you have all the required dependencies:
```

Comment:
Should we also mention the optional dependencies of shfmt and shellcheck?

Reply:
Good idea. I added some more info about that.